services/registrations/handler.js

```diff
@@ -217,13 +226,13 @@ export async function sendEmail(user, existingEvent, registrationStatus, id) {
     let tempId = "d-11d4bfcbebdf42b686f5e7d0977aa952";
     let tempCalendarId = "d-b517e4c407e4421a8886140caceba551";
 
-    if (registrationStatus === "cancelled") {
+    if (registrationStatus == "cancelled") {
```

Comment:
is there a reason why we changed these from strict equals to ==?

Reply:
No reason. can change it back, I'm not sure why my github like defaulted into recommending all those === to == changes. Should I change it back?